lib/TransferManager.cs

```diff
@@ -314,7 +314,7 @@ public static Task<TransferStatus> UploadDirectoryAsync(string sourcePath, Cloud
         {
             DirectoryLocation sourceLocation = new DirectoryLocation(sourcePath);
             AzureBlobDirectoryLocation destLocation = new AzureBlobDirectoryLocation(destBlobDir);
-            FileEnumerator sourceEnumerator = new FileEnumerator(sourceLocation);
+            FileEnumerator sourceEnumerator = new FileEnumerator(sourceLocation, null != options? options.FollowSymlink : false);
```

Comment:
Minor: Format. Need additional blank.

Reply:
Thanks for your comments. Format issue will be fixed in another PR.